src/eos-google-chrome

```diff
@@ -94,6 +140,12 @@ class GoogleChromeLauncher:
         except OSError as e:
             exit_with_error("Could not launch Chrome: {}".format(repr(e)))
 
+        self._wait_for_installation()
```

Comment:
First of all - what's _FBE_?

Thank you for the feedback. I agree with the points that @mariospr proposed. That's how I see it in terms of modules/repos:

eos-google-chrome-commons (?)
- just a Python module with a `check_chrome_installation` function, returning just a boolean

eos-google-chrome-installer
- autostarted if `$HOME/.config/google-chrome-installation-done` doesn't exist or some gsetting is not set
- on the beginning of the script, we check internet connectivity - if we don't have it, we just stop execution (because we are fine with doing nothing in that point, right? or do we need to install Chrome automatically as soon as user gets an access to the internet?)
- runs Gnome Software in headless-app mode, installs Chrome automatically, watches dbus signals till it will receive info about successful Chrome installation
- sets Chrome as a default browser (xdg-* stuff)
- creates `google-chrome-installation-done` file

eos-google-chrome-helper
- does the same as currently, but uses _commons_ module to check whether Chrome is installed

Do you agree? Did I get your suggestions correctly?